2024-11-13 02:24:50,789 - app - ERROR - Internal Server Error: 500 Internal Server Error: The server encountered an internal error and was unable to complete your request. Either the server is overloaded or there is an error in the application.
2024-11-13 02:41:21,918 - app - ERROR - Exception on /forgotPassword/forgot [GET]
Traceback (most recent call last):
  File "/home/josemorgado/Repo/RaboDeToro-hub-1/RaboDeToroHub1venv/lib/python3.12/site-packages/flask/app.py", line 1473, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/josemorgado/Repo/RaboDeToro-hub-1/RaboDeToroHub1venv/lib/python3.12/site-packages/flask/app.py", line 882, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/josemorgado/Repo/RaboDeToro-hub-1/RaboDeToroHub1venv/lib/python3.12/site-packages/flask_restful/__init__.py", line 298, in error_router
    return original_handler(e)
           ^^^^^^^^^^^^^^^^^^^
  File "/home/josemorgado/Repo/RaboDeToro-hub-1/RaboDeToroHub1venv/lib/python3.12/site-packages/flask/app.py", line 880, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/josemorgado/Repo/RaboDeToro-hub-1/RaboDeToroHub1venv/lib/python3.12/site-packages/flask/app.py", line 865, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/josemorgado/Repo/RaboDeToro-hub-1/core/decorators/decorators.py", line 14, in decorated_function
    return f(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^
  File "/home/josemorgado/Repo/RaboDeToro-hub-1/app/modules/forgotPassword/routes.py", line 32, in forgot
    return render_template("resetPassword/forgotPasswordRequest.html")
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/josemorgado/Repo/RaboDeToro-hub-1/RaboDeToroHub1venv/lib/python3.12/site-packages/flask/templating.py", line 150, in render_template
    return _render(app, template, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/josemorgado/Repo/RaboDeToro-hub-1/RaboDeToroHub1venv/lib/python3.12/site-packages/flask/templating.py", line 131, in _render
    rv = template.render(context)
         ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/josemorgado/Repo/RaboDeToro-hub-1/RaboDeToroHub1venv/lib/python3.12/site-packages/jinja2/environment.py", line 1304, in render
    self.environment.handle_exception()
  File "/home/josemorgado/Repo/RaboDeToro-hub-1/RaboDeToroHub1venv/lib/python3.12/site-packages/jinja2/environment.py", line 939, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "/home/josemorgado/Repo/RaboDeToro-hub-1/app/modules/forgotPassword/templates/resetPassword/forgotPasswordRequest.html", line 1, in top-level template code
    {% extends "base_template.html" %}
    ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/josemorgado/Repo/RaboDeToro-hub-1/app/templates/base_template.html", line 304, in top-level template code
    {% block scripts %}{% endblock %}
    ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/josemorgado/Repo/RaboDeToro-hub-1/app/modules/forgotPassword/templates/resetPassword/forgotPasswordRequest.html", line 42, in block 'scripts'
    <script src="{{ url_for('forgotPassword.assets', subfolder='js', filename='scripts.js') }}"></script>
    ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/josemorgado/Repo/RaboDeToro-hub-1/RaboDeToroHub1venv/lib/python3.12/site-packages/flask/app.py", line 1084, in url_for
    return self.handle_url_build_error(error, endpoint, values)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/josemorgado/Repo/RaboDeToro-hub-1/RaboDeToroHub1venv/lib/python3.12/site-packages/flask/app.py", line 1073, in url_for
    rv = url_adapter.build(  # type: ignore[union-attr]
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/josemorgado/Repo/RaboDeToro-hub-1/RaboDeToroHub1venv/lib/python3.12/site-packages/werkzeug/routing/map.py", line 924, in build
    raise BuildError(endpoint, values, method, self)
werkzeug.routing.exceptions.BuildError: Could not build url for endpoint 'forgotPassword.assets' with values ['filename', 'subfolder']. Did you mean 'forgotPassword.scripts' instead?
2024-11-13 02:41:21,922 - app - ERROR - Internal Server Error: 500 Internal Server Error: The server encountered an internal error and was unable to complete your request. Either the server is overloaded or there is an error in the application.
2024-11-13 03:19:00,568 - app - ERROR - Exception on /forgotPassword/forgot [GET]
Traceback (most recent call last):
  File "/home/josemorgado/Repo/RaboDeToro-hub-1/RaboDeToroHub1venv/lib/python3.12/site-packages/flask/app.py", line 1473, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/josemorgado/Repo/RaboDeToro-hub-1/RaboDeToroHub1venv/lib/python3.12/site-packages/flask/app.py", line 882, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/josemorgado/Repo/RaboDeToro-hub-1/RaboDeToroHub1venv/lib/python3.12/site-packages/flask_restful/__init__.py", line 298, in error_router
    return original_handler(e)
           ^^^^^^^^^^^^^^^^^^^
  File "/home/josemorgado/Repo/RaboDeToro-hub-1/RaboDeToroHub1venv/lib/python3.12/site-packages/flask/app.py", line 880, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/josemorgado/Repo/RaboDeToro-hub-1/RaboDeToroHub1venv/lib/python3.12/site-packages/flask/app.py", line 865, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/josemorgado/Repo/RaboDeToro-hub-1/core/decorators/decorators.py", line 14, in decorated_function
    return f(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^
  File "/home/josemorgado/Repo/RaboDeToro-hub-1/app/modules/forgotPassword/routes.py", line 28, in forgot
    return render_template("resetPassword/forgotPasswordRequest.html")
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/josemorgado/Repo/RaboDeToro-hub-1/RaboDeToroHub1venv/lib/python3.12/site-packages/flask/templating.py", line 150, in render_template
    return _render(app, template, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/josemorgado/Repo/RaboDeToro-hub-1/RaboDeToroHub1venv/lib/python3.12/site-packages/flask/templating.py", line 131, in _render
    rv = template.render(context)
         ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/josemorgado/Repo/RaboDeToro-hub-1/RaboDeToroHub1venv/lib/python3.12/site-packages/jinja2/environment.py", line 1304, in render
    self.environment.handle_exception()
  File "/home/josemorgado/Repo/RaboDeToro-hub-1/RaboDeToroHub1venv/lib/python3.12/site-packages/jinja2/environment.py", line 939, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "/home/josemorgado/Repo/RaboDeToro-hub-1/app/modules/forgotPassword/templates/resetPassword/forgotPasswordRequest.html", line 1, in top-level template code
    {% extends "base_template.html" %}
    ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/josemorgado/Repo/RaboDeToro-hub-1/app/templates/base_template.html", line 200, in top-level template code
    {% block content %}{% endblock %}
    ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/josemorgado/Repo/RaboDeToro-hub-1/app/modules/forgotPassword/templates/resetPassword/forgotPasswordRequest.html", line 11, in block 'content'
    <form action="{{ url_for('forgotPassword.resetPassword') }}" method="post">
    ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/josemorgado/Repo/RaboDeToro-hub-1/RaboDeToroHub1venv/lib/python3.12/site-packages/flask/app.py", line 1084, in url_for
    return self.handle_url_build_error(error, endpoint, values)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/josemorgado/Repo/RaboDeToro-hub-1/RaboDeToroHub1venv/lib/python3.12/site-packages/flask/app.py", line 1073, in url_for
    rv = url_adapter.build(  # type: ignore[union-attr]
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/josemorgado/Repo/RaboDeToro-hub-1/RaboDeToroHub1venv/lib/python3.12/site-packages/werkzeug/routing/map.py", line 924, in build
    raise BuildError(endpoint, values, method, self)
werkzeug.routing.exceptions.BuildError: Could not build url for endpoint 'forgotPassword.resetPassword'. Did you forget to specify values ['token']?
2024-11-13 03:19:00,574 - app - ERROR - Internal Server Error: 500 Internal Server Error: The server encountered an internal error and was unable to complete your request. Either the server is overloaded or there is an error in the application.
